radlibDemoApp/plugins/com.radlib.cordova.rfidreader/www/ReaderTSL_1128_UHF.js

```diff
@@ -1,5 +1,52 @@
 
-var exec = require('cordova/exec');
-var tsl_1128_uhf;
+var bluetooth = require('./CommBluetooth');
+var ReaderTSL_1128_UHF = {};
 
-module.exports = tsl_1128_uhf;
```

Comment:
This goes at the very bottom of the file after all your functions. Move it to the bottom